src/app/ui/team-page/team-applications/team-applications.component.ts

```diff
@@ -66,7 +64,7 @@ export class TeamApplicationsComponent implements OnInit {
     We are sorry to inform you that you have been kicked out from your current team
     </p>
     <p>
-    Visit your dashboard to join another team.
+    Visit your <a href='https://leaderboard-79b77.firebaseapp.com/profile'>dashboard</a> to join another team.
```

Comment:
please don't hardcode this and use `environment.firebase.authDomain` from `src/environments/environment`